assets/keymaps/default-linux.json

```diff
@@ -50,6 +52,7 @@
     "context": "Editor",
     "bindings": {
       "escape": "editor::Cancel",
+      "cancel": "editor::Cancel",
```

Comment:
Do you have a physical `cancel` key on your keyboard?
I've never seen this and it feels pretty niche.

Reply:
I do not. The ones I currently have (with a custom layout, not physically on the keyboard) are Cut, Copy, Paste, Undo, and Redo. But I tried to be exhaustive with this PR.

Reply:
Similar question about `execute` and `clear` keys.

Reply:
I'd say let's only create default mapping for cut/copy/paste/new/open/save as these are not uncommon on Multimedia keyboards.

Let's hold off and not map `cancel`, `execute` and `clear`; they feel a little niche (and we can't test them).  Anyone who wants these should be able to easily map them themselves.

Reply:
I’ve removed Execute, Cancel, and Clear.